### DISCUSSION TITLE
Daily-Semana 24/10 a 29/10

### DISCUSSION THREAD

**Discussion by Paulac23** [Paulac23/truwebque#35] [General]
En esta sección, vamos a comunicarnos para realizar la Daily. Va a estar dividida por semana, hasta finalizar el Sprint 2.
La idea es comentar nuestro trabajo para estar al tanto del avance del equipo. 
Para continuar un orden y sentido, propongo las siguientes secciones que detallaremos en esta discusión (no son necesarias utilizarlas todas)
Por ejemplo: 
-Estoy trabajando en:..........
-Pude finalizar:..........
-Voy a continuar:.............
-Bloqueantes: (Adjuntar dudas o preguntas hacia los demás miembros, que nos "bloquean" e impiden avanzar en mis tareas)

**Comment by Paulac23** [comment]
- Estoy trabajando en https://trello.com/c/C8C2GTWJ/1-corregir-el-responsive-del-carrusel-del-indexhtml

1. Pude finalizar: diseñar las imagenes para el carrusel, corregir el responsive
2. Voy a continuar: corrigiendo los colores y ubicar los botones de forma que no tapen la imagen, para después compartirlo.

**Reply by agusval** [reply]
Tengo alternativas de paletas de colores, voy a probarlas dentro de CSS para ver el resultado concreto. Quizas pueda servir.
